src/android/src/notification/NotificationPlugin.java

```diff
@@ -0,0 +1,217 @@
+package com.davinci.studio.notification;
+
+import android.app.Activity;
+import android.app.Notification;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+
+import org.apache.cordova.CordovaWebView;
+import org.apache.cordova.CallbackContext;
+import org.apache.cordova.CordovaPlugin;
+import org.apache.cordova.CordovaInterface;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.json.JSONException;
+
+import java.util.ArrayList;
+
+public class NotificationPlugin extends CordovaPlugin {
+  private static final int NOTIFICATION_ID = 1;
+
+  private Activity mActivity;
+  private NotificationManager mNotifyManager;
+  private Notification.Builder mBuilder;
+  private NotificationConfig mNotificationConfig;
+  
+  public NotificationPlugin() {}
+
+  public void initialize(CordovaInterface cordova, CordovaWebView webView) {
+    super.initialize(cordova, webView);
+
+    mActivity = cordova.getActivity();
+    mNotifyManager = (NotificationManager) mActivity.getSystemService(Context.NOTIFICATION_SERVICE);
+    mBuilder = new Notification.Builder(mActivity);
+    mNotificationConfig = new NotificationConfig();
+  }
+
+  public boolean execute(final String action, JSONArray args, CallbackContext callbackContext) throws JSONException {
+    if (action.equals("setTitle")) {
+      String title = args.getString(0);
+
+      if (title != "null") {
+        this.setTitle(title);
+
+        return true;
+      }
+
+      this.setTitle();
+
+      return true;
+    }
+
+    if (action.equals("setText")) {
+      String text = args.getString(0);
+
+      if (text != "null") {
+        this.setText(text);
+
+        return true;
+      }
+
+      this.setText();
+
+      return true;
+    }
+
+    if (action.equals("setTicker")) {
+      if (args.getString(0) != "null") {
```

Comment:
analogicznie jak w title, poza tym tu masz powtorzenie wyciągnięcia ``args.getString(0)``, w innych przypadkach przypisujesz to do zmiennej